src/main/kotlin/macgram/blackjackbackend/domain/user/Player.kt

```diff
@@ -0,0 +1,83 @@
+package macgram.blackjackbackend.domain.user
+
+import macgram.blackjackbackend.domain.card.Card
+import macgram.blackjackbackend.domain.card.HoldCards
+import java.math.BigDecimal
+
+class Player : Gamer {
+    val MIN_MONEY = 0L
+    val MAX_SUM = 21L
+    val cards : HoldCards = HoldCards()
+
+    var money : BigDecimal = BigDecimal.ZERO
+    var battingMoney : BigDecimal = BigDecimal.ZERO
+    var continueFlag : Boolean = true
+
+    fun Player(money : Long) {
+        this.money = BigDecimal.valueOf(money)
+    }
+
+    fun batting(batting : Long) {
+        validateBatting(batting)
+        this.battingMoney = BigDecimal.valueOf(batting)
+    }
+
+    fun validateBatting(money : Long) {
+        if (money == MIN_MONEY) {
+            throw IllegalArgumentException("배팅 금액은 0원 이상 이여야 합니다.")
+        }
+
+        if (this.money.toLong() - money < MIN_MONEY) {
+            throw IllegalArgumentException("배팅 금액이 부족합니다.")
+        }
+    }
+
+    override fun addCard(card : Card) {
+        if (this.continueFlag) {
+            this.cards.add(card)
+        }
+
+        if (calculate() >= MAX_SUM) {
+            this.continueFlag = false
+        }
+    }
+
+    override fun calculate() : Int {
+        return this.cards.calculate()
+    }
+
+    fun checkContinued(isContinued : Boolean) {
+        this.continueFlag = isContinued
+    }
```

Comment:
파라미터 명을 동사로 시작하는 것은 지양해주시면 좋습니다.
is의 경우에는 예외적으로 변수명으로 쓰이기도 하나, Boolean 형 변수에 일반적으로 쓰이는 변수명은 '형용사'입니다.

```suggestion
    fun checkContinuable(continuable : Boolean) {
        this.continueFlag = continuable 
    }
```

또 한 가지 말씀드리면,
위 메서드는 사실 setContinueFlag가 하는 일을 정확히 똑같이 하고 있습니다.
이 메서드에 check라는 이름은 어울리지 않고,
위쪽 리뷰대로 생성자를 변경할 경우 메서드가 중복되니
해당 메서드를 제거하는 게 좋을 것 같습니다.